### DISCUSSION TITLE
Another New User... Trying to utilize STL import

### DISCUSSION THREAD

**Discussion by retroglyphic** [zalo/CascadeStudio#88] [Q&A]
Hi,

I was referred to Cascade Studio by Gilboonet. I've been a user of openSCAD, amongst other 3D tools. I've made a quick SCAD script that allows a user to import their model and it will effectively chamfer the bottom by user defined parameters. This is all well and good for users who have openSCAD installed on their machine. But now I am trying to make something similar for those who would like an online tool. 

CascadeStudio seems promising for this. However, I am struggling with the use of import STL. I am using ASCII stl files. When I try to import a very simple stl, the console reports >Test.stl loaded successfully! Converting to OCC now... 

What I have learned is that models might take awhile to convert. I am currently working on a model and it has been converting for over 60 minutes. I am curious if this is normal behavior? 

Thank you

Steve

**Comment by raydeleu** [comment]
I just tried this with a STEP file of 556 kB. It loaded almost instantly. However, be aware that you have to add a line to your code to render your new part. For example: 

sceneShapes.push(externalShapes['honeycomb.step']);

**Comment by raydeleu** [comment]
![Screen Shot 2021-05-17 at 16 16 29](https://user-images.githubusercontent.com/38007983/118503879-4db33900-b72b-11eb-97b3-d1e1a102f812.png)

Did the same with an STL file. Took a little longer but still relatively fast. Compared to STEP this model seems to contain a lot more edges, maybe that the number of edges in your model is even larger.
